server.py

```diff
@@ -0,0 +1,40 @@
+import grpc
+import hwsc_file_transaction_svc_pb2_grpc
+from concurrent import futures
+from enum import Enum
+from file_transaction_service import FileTransactionService
+
+
+class State(Enum):
+    """A current state class of file transaction service"""
+    AVAILABLE = 0
+    UNAVAILABLE = 1
+
+
+class Server:
```

Comment:
This is the class or object from https://github.com/hwsc-org/hwsc-file-transaction-svc/pull/44/files#diff-5bc02cefb3ea9e27f1a6776eabd1935dR5